app/api/domains/cho.py

```diff
@@ -428,7 +429,7 @@ async def handle(self, player: Player) -> None:
             t_chan.send(msg, sender=player)
 
         player.update_latest_activity_soon()
-        log(f"{player} @ {t_chan}: {msg}", Ansi.LCYAN, file=".data/logs/chat.log")
+        log(f"{player} @ {t_chan}: {msg}", Ansi.LCYAN)
```

Comment:
This is now replaced by the flexibility of logger configuration -- you can now send logs to not only standard streams, but a file, tcp/udp socket, or anything else you desire.